CMakeLists.txt

```diff
@@ -54,6 +54,10 @@ else()
   message("Doing a local build")
 endif()
 
+if($ENV{NUPIC_ARM_BUILD})
```

Comment:
@pettitda I agree, the CMAKE_SYSTEM_PROCESSOR option is a better way to detect and target ARM based systems such as RPI3.